content/en/platform/corda/1.6/cenm/cenm-support-matrix.md

```diff
@@ -53,9 +53,9 @@ CENM currently supports the following databases:
 
 ## JDK support
 
-Corda Enterprise Edition 4.10+ has been tested and verified to work with **Oracle JDK 8 JVM 8u352** and **Azul Zulu Enterprise 8u352**. Corda Enterprise Network Manager 1.5.9+ has been tested and verified to work with **Oracle JDK 8 JVM 8u382** and **Azul Zulu Enterprise 8u382**. For Azure deployment downloadable from [Azul Systems](https://www.azul.com/downloads/azure-only/zulu/).
+Corda Enterprise Edition 4.11 has been tested and verified to work with **Oracle JDK 8 JVM 8u381** and **Azul Zulu Enterprise 8u382**. Corda Enterprise Network Manager 1.6 has been tested and verified to work with **Oracle JDK 8 JVM 8u392** and **Azul Zulu Enterprise 8u392**. For Azure deployment downloadable from [Azul Systems](https://www.azul.com/downloads/azure-only/zulu/).
```

Comment:
>Corda Enterprise Edition 4.11+ and the Corda Enterprise Network Manager 1.6+ have been tested and verified to work with **Oracle JDK 8 JVM 8u382** and **Azul Zulu Enterprise 8u382**. For Azure deployment downloadable from [Azul Systems](https://www.azul.com/downloads/azure-only/zulu/).

Both are now same JDK again (like before the initial change to 382 in cenm) so I changed the version back to what it was before. Also it was CRAFT tested against 382 so docs should reflect that, (not 392)

Reply:
Sorry, it should be Oracle 381

Reply:
Updated